src/services/storage/storage-factories.ts

```diff
@@ -0,0 +1,17 @@
+import { networks } from '../../constants/networks'
+import { Network } from '../../model/storage/network.model'
+import { Swarm } from '../../model/storage/swarm.model'
+
+export function networkFactory(): Network {
+  return Object.assign({}, networks[0])
```

Comment:
please use this notation at other places as well.
```suggestion
  return { ...networks[0] }
```